.github/workflows/build-v3.yml

```diff
@@ -0,0 +1,87 @@
+name: build v3
```

Comment:
The v3 pipeline is [here](https://github.com/SAP/cloud-sdk-js/blob/main/.github/workflows/build.yml), which was created already.
I was not able to double check, when I was outside today.
Sorry for confusing.
You can just touch the node versions there.

Reply:
I know this builds v3 but it also builds the matrix of node 14,16,18 but we only want node 18